### DISCUSSION TITLE
CM version and Docker

### DISCUSSION THREAD

**Discussion by mrmuiz** [007revad/ContainerManager_for_all_armv8#31] [Q&A]
Does this version work ? :  https://global.synologydownload.com/download/Package/spk/ContainerManager/24.0.2-1525/ContainerManager-armv8-24.0.2-1525.spk

And after CM  is installed. Does it come with Docker? If yes is that version that Synology offers? And how can i update that to latest version?

**Comment by 007revad** [comment]
That's the beta version of container manager. Some people have lost all their containers and data when upgrading to the beta version.

If you wanted to ty it you'd need to change lines 427 to 431 in the script from: 
```
# Install Container Manager
package_install ContainerManager "$targetvol" "Container Manager"
if [[ $install_from_server_failed == "yes" ]]; then
    do_manual_install
fi
```
to:
```
# Install Container Manager
    do_manual_install
```
Then do a Manual Install of your downloaded spk file when the scripts tells you to.

<br>

Container Manager includes docker. The docker version is in the Container Manager version version number.
- Container Manager 20.10.23 includes Docker version 20.10.23
- Container Manager Beta 24.0.2 includes Docker version 24.0.2

As far as I know you can't easily update the Docker version on a Synology.

**Reply by mrmuiz** [reply]
Thx. And update can be done with this: https://github.com/telnetdoogie/synology-docker but not this time. Only official CM/Docker  he says.
